ReplayKitExample/ReplayKitExample/Telecine.swift

```diff
@@ -0,0 +1,228 @@
+//
+//  Telecine.swift
+//  ReplayKitExample
+//
+//  Created by Chris Eagleston on 8/10/19.
+//  Copyright © 2019 Twilio. All rights reserved.
+//
+
+import Foundation
+import CoreMedia
+import CoreVideo
+
+enum TelecineResult {
+    case dropFrame
+    case deliverFrame
+}
+
+protocol InverseTelecine {
+    func process(input: CMSampleBuffer, last: CMSampleBuffer) -> (TelecineResult, CMTime)
+}
+
+class InverseTelecine60p : InverseTelecine {
```

Comment:
1. Add header doc for each Telecine implementation
2. Remove log lines, or if some are absolutely needed wrap in debug definitions